### DISCUSSION TITLE
Horizontal tabs aren't hided in LibreWolf 99-01, Linux Mint, Cinnamon!

### DISCUSSION THREAD

**Discussion by ExpertMen** [piroor/treestyletab#3108] [General]
## Short description
This css for hiding horizontal tabs doesn't work in LibreWolf 99-01. - https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-horizontal-tabs-at-the-top-of-the-window-1349-1672-2147
## Steps to reproduce

 1. Start LibreWolf 99-01 with clean profile.
 2. Install TST.
 3. Follow this tut (open about:profiles, open your profile directory, etc.) (https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-horizontal-tabs-at-the-top-of-the-window-1349-1672-2147)
 4. Make sure that changes is saved, and restart LibreWolf. See? Didn't worked!

## Expected result
Horizontal tabs are hide in LibreWolf 99-01.

## Actual result
They don't hided.

## Environment

 * Platform (OS): Linux Mint Cinnamon
 * Version of LibreWolf: 99-01
 * Version (or revision) of Tree Style Tab: latest (3.8.21)

**Comment by piroor** [comment]
(I've moved this from issues to discussions, because it is out of scope of TST project itself. It is a common knowledge about customizing Firefox with userChrome.css.)

**Comment by irvinm** [comment]
The instructions worked fine for me.  (LM 20.3, LibreWolf 99.0.1)  Maybe you should list out exactly what you did.  I don't use Linux very often, but did find it challenging to find the "Root Directory" of LibreWolf ... I couldn't find it until I read:  [Where do I find my librewolf.overrides.cfg?](https://librewolf.net/docs/settings/#where-do-i-find-my-librewolfoverridescfg)

In Firefox, simply doing about:profiles -> default profile -> Root Directory -> Open Directory works fine.

![image](https://user-images.githubusercontent.com/979729/164253004-e730a661-611f-4390-b7e8-3fa70cb3cb42.png)

**Reply by ExpertMen** [reply]
Thank you @irvinm , that works for me! Btw, I am using LM 20.3 too :))

**Reply by irvinm** [reply]
Glad to hear you were able to get it to work!  Please go ahead and mark this as "answered" please.

**Reply by ExpertMen** [reply]
idk how do I do it....

**Reply by irvinm** [reply]
Oh, never mind ... this was opened as "General" not "Q&A". :)
